### PR TITLE
Use single camera

### LINKED ISSUE #13: Remove dual camera dependency
I think it's fair to say we'll usually be using only a single camera. We should remove the dual camera from this repository and use yearly vision repositories instead, using this only as a base, updating as necessary.

### PR DESCRIPTION
**Pull request by JacksonCoder**
I removed the `Vision.front_camera` members and `Vision.rear_camera` members and replaced them with a single object `camera`. This reads from the first camera (index 0), and doesn't require any nt_utils.

Closes #13

### CODE REVIEW

File: vision/nt_utils.py [hunk 1]
```diff
@@ -59,4 +59,4 @@ def get_number(key):
 			print("[NetworkTable] not connected")
 	except:
 		return 0
-		print "NetworkTable error getting number"
+		print("NetworkTable error getting number")
```
Comment (andrewda) [inline vision/nt_utils.py]: Do this in another PR please?

Comment (andrewda) [reply vision/nt_utils.py]: Nevermind you just used the same branch as the last one. I'll update.

File: vision/app.py [hunk 2]
```diff
@@ -80,8 +80,7 @@ def run_image(self):
 			cv2.destroyAllWindows()
 
 	def run_video(self):
-		cameraFront = WebcamVideoStream(src=self.source[1]).start()
-		cameraRear = WebcamVideoStream(src=self.source[0]).start()
+		camera = WebcamVideoStream(src=self.source[0]).start()
```
Comment (andrewda) [inline vision/app.py]: `src=self.source` and change `sources` to `source` in [utils.py](https://github.com/SouthEugeneRoboticsTeam/vision/blob/master/vision/utils.py#L17), along with changing the type to `int`, the default to `0`, and the help text to `video sources (default=0)`.

**Comment by andrewda** [comment]
Also, please make sure to update your local branch before adding anything more -- I made a few changes to update it to master. My requested change above still applied even if it's collapsed.

**Comment by JacksonCoder** [comment]
Pushed requested changes.
